packages/components/table/src/filter-panel.vue

```diff
@@ -133,6 +134,10 @@ export default defineComponent({
     upDataColumn: {
       type: Function,
     },
+    appendTo: {
+      type: String,
+      default: 'body',
```

Comment:
Why do you declare the default value to body ? Previously, it would append to` <div id="el-popper-container-xxx"></div>`.

Reply:
Sorry, I made this PR a long time ago so I don't remember, could it be that at that point it was `body` instead of `<div id="el-popper-container-xxx"></div>` and I used `body` because of that? I can try changing it

Reply:
Perhaps the original logical structure should not be changed by default. If the developer needs to make custom adjustments, they can just pass the parameters themselves.

Reply:
Removed the default value